inc/saivlan.h

```diff
@@ -657,7 +821,10 @@ typedef struct _sai_vlan_api_t
     sai_get_vlan_stats_fn               get_vlan_stats;
     sai_get_vlan_stats_ext_fn           get_vlan_stats_ext;
     sai_clear_vlan_stats_fn             clear_vlan_stats;
-
+    sai_create_vlan_stack_fn            create_vlan_stack;
```

Comment:
vlan stack/translation is a different stage in the pipeline.
It involves tag manipulations only. 

Should we be modifying the sai_vlan_api_t ?

Instead Is it better to add a new API sai_vlan_stack_api_t and add 
the vlan_stack related functions there ?

Reply:
It is also an option. But if I do so, the sai_vlan_stack_api_t should be put in another header file, right?

Reply:
yes that would be another header file. However since we this is another stage in the pipeline it should be fine.

Reply:
Hi @srj102 ,
I have split the vlan stack into another header file. Please check if it helps.